containers/go/.devcontainer/library-scripts/meta.env

```diff
@@ -0,0 +1 @@
+VERSION='dev'
```

Comment:
Is this something that be put into a shared spot/created programatically, so in the future we don't have to change it in each directory (if it is already automatically created?)

Reply:
It actually is programmatically updated on build [here](https://github.com/microsoft/vscode-dev-containers/blob/86fc18bd7cdeee51db6284db039e3d6ecfcc3403/build/src/prep.js#L248) using a handlebars template. This is just a stub that is there to get the "COPY" statement to execute in all cases when you're working on the definition outside of the build system (e.g. open the folder using Remote - Containers).

`COPY` will bomb if it doesn't find any files to copy annoyingly even if you use `*`.  There's a similar trick with a `noop.txt` file in the anaconda and miniconda definitions. It's particularly important for definitions like `cpp` that do not have a library-scripts folder otherwise. Anything could be in this file really. I just added the dev version so you'd see something in the `devcontainer-info` command but technically it could be empty.

The actual build script injects a meta.env that overwrites this, so it's not used for any other purpose.

Reply:
We could remove it everywhere from anywhere that already has a `library-scripts` folder, but I went for consistency.